server/src/test/java/com/linecorp/centraldogma/server/internal/api/TokenServiceTest.java

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright 2018 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package com.linecorp.centraldogma.server.internal.api;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+import java.util.Collection;
+
+import org.junit.BeforeClass;
+import org.junit.ClassRule;
+import org.junit.Test;
+
+import com.linecorp.armeria.server.HttpStatusException;
+import com.linecorp.centraldogma.common.Author;
+import com.linecorp.centraldogma.server.internal.admin.authentication.User;
+import com.linecorp.centraldogma.server.internal.metadata.MetadataService;
+import com.linecorp.centraldogma.server.internal.metadata.MigrationUtil;
+import com.linecorp.centraldogma.server.internal.metadata.Token;
+import com.linecorp.centraldogma.testing.internal.ProjectManagerRule;
+
+import io.netty.util.internal.StringUtil;
+
+public class TokenServiceTest {
+
+    @ClassRule
+    public static final ProjectManagerRule rule = new ProjectManagerRule() {
+        @Override
+        protected void afterExecutorStarted() {
+            MigrationUtil.migrate(projectManager(), executor());
+        }
+    };
+
+    private static final Author adminAuthor = new Author("admin@localhost.com");
+    private static final Author guestAuthor = new Author("guest@localhost.com");
+    private static final User admin = new User("admin@localhost.com", User.LEVEL_ADMIN);
+    private static final User guest = new User("guest@localhost.com");
+
+    private static TokenService tokenService;
+
+    @BeforeClass
+    public static void beforeClass() {
+        tokenService = new TokenService(rule.projectManager(), rule.executor(),
+                                        new MetadataService(rule.projectManager(), rule.executor()));
+    }
+
+    @Test
+    public void adminToken() {
+        assertThat(tokenService.createToken("forAdmin1", true, adminAuthor, admin)
+                               .toCompletableFuture().join().object().isActive()).isTrue();
```

Comment:
I'm curious if it's a good idea to return a `CompletableFuture` instead of a `CompletionStage` so that the tests don't need to convert it.

Reply:
Thanks! I will address this comment in the next PR! :-)

Reply:
Thanks, @hyangtack !